[2023-01-22T03:34:13.621+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: debts_tracker_etl.create_encrypt_file manual__2023-01-22T03:34:08.266733+00:00 [queued]>
[2023-01-22T03:34:13.632+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: debts_tracker_etl.create_encrypt_file manual__2023-01-22T03:34:08.266733+00:00 [queued]>
[2023-01-22T03:34:13.633+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-01-22T03:34:13.634+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 2
[2023-01-22T03:34:13.635+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-01-22T03:34:13.647+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): create_encrypt_file> on 2023-01-22 03:34:08.266733+00:00
[2023-01-22T03:34:13.654+0000] {standard_task_runner.py:55} INFO - Started process 760 to run task
[2023-01-22T03:34:13.658+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'debts_tracker_etl', 'create_encrypt_file', 'manual__2023-01-22T03:34:08.266733+00:00', '--job-id', '30', '--raw', '--subdir', 'DAGS_FOLDER/debts_tracker_etl.py', '--cfg-path', '/tmp/tmpjwb7uzup']
[2023-01-22T03:34:13.662+0000] {standard_task_runner.py:83} INFO - Job 30: Subtask create_encrypt_file
[2023-01-22T03:34:13.792+0000] {task_command.py:388} INFO - Running <TaskInstance: debts_tracker_etl.create_encrypt_file manual__2023-01-22T03:34:08.266733+00:00 [running]> on host 4b1c480b6ccb
[2023-01-22T03:34:13.881+0000] {taskinstance.py:1507} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Tony Stark, Peter Parker
AIRFLOW_CTX_DAG_ID=debts_tracker_etl
AIRFLOW_CTX_TASK_ID=create_encrypt_file
AIRFLOW_CTX_EXECUTION_DATE=2023-01-22T03:34:08.266733+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-01-22T03:34:08.266733+00:00
[2023-01-22T03:34:13.913+0000] {logging_mixin.py:137} INFO - 0-----
[2023-01-22T03:34:13.913+0000] {logging_mixin.py:137} INFO - b'gAAAAABjzK618GjSU7FR5PlAEwjNeWlfZwATTsjktnBl98WjWUsq_zIPek7o4zKfmxAZvE5SXk4rQYVOvMSl2VajaKPerwgg9-vU7ygGPkm2-lm08ZnXImm6mzC2FgPoTe9y55OVn2DKt5QMQdsNr-5wtDnSQHZxoi4TwGG6s7l-qlr2evdXGHl2zmAXuYc1m1z7fvZiN1BslPFb29WIja8x4O-A0ubScQd3TfbVonvjb4xVkAGUnXaueeIv_LicAxsQPlU-_3EeyQmAhFwm3i0cyrwcpqXAIWavAdo8ESRJV0EVG2eypOX0pip-VSgr5PIQWgDAKWmTeMU_FILBxn-jKK6oawLes3p6q_I9Xx3-Z6cNZ83GJ6k='
[2023-01-22T03:34:13.914+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-01-22T03:34:13.926+0000] {taskinstance.py:1318} INFO - Marking task as SUCCESS. dag_id=debts_tracker_etl, task_id=create_encrypt_file, execution_date=20230122T033408, start_date=20230122T033413, end_date=20230122T033413
[2023-01-22T03:34:13.954+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-01-22T03:34:13.976+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
